internal/v1/handler_compose_image.go

```diff
@@ -126,7 +127,10 @@ func (h *Handlers) handleCommonCompose(ctx echo.Context, composeRequest ComposeR
 			return ComposeResponse{}, err
 		}
 	} else {
-		repositories = buildRepositories(arch, composeRequest.ImageRequests[0].ImageType)
```

Comment:
Could we drop buildRepositories?

Reply:
sure, I removed it while trying to keep the sanity check